imports/api/activities/activity.methods.js

```diff
@@ -37,51 +37,43 @@ Meteor.methods({
       throw new Meteor.Error('Image is required for public activities');
     }
 
-    const resourceIndex = formValues.resource.resourceIndex;
-
     try {
-      const add = Activities.insert(
-        {
-          host,
-          authorId: user._id,
-          authorName: user.username,
-          title: formValues.title,
-          subTitle: formValues.subTitle || null,
-          longDescription: formValues.longDescription,
-          resource: formValues.resource.label || null,
-          resourceId: formValues.resource._id,
-          resourceIndex,
-          place: formValues.place || null,
-          practicalInfo: formValues.practicalInfo || null,
-          internalInfo: formValues.internalInfo || null,
-          address: formValues.address || null,
-          capacity: formValues.capacity || 20,
-          datesAndTimes: formValues.datesAndTimes,
-          imageUrl: uploadedImage || null,
-          isSentForReview: false,
-          isPublicActivity: formValues.isPublicActivity,
-          isActivitiesDisabled: formValues.isActivitiesDisabled,
-          isPublished: true,
-          creationDate: new Date(),
-        },
-        () => {
-          if (!formValues.isPublicActivity) {
-            return;
-          }
-          Meteor.call('createChat', formValues.title, add, (error, result) => {
-            if (error) {
-              Logger.createLogger(
-                `Chat is not created due to error: ${
-                  error.reason || error.error
-                }`
-              );
-              throw new Meteor.Error('Chat is not created');
-            }
-          });
-        }
-      );
+      const add = Activities.insert({
+        host: host,
```

Comment:
futher more, now we have schemas so, we should define default values under schemas defaultValue prop